### ISSUE TITLE
Issue to trigger new releases

### ISSUE THREAD

**Issue by ablaom** [JuliaAI/IterationControl.jl#20]
@JuliaRegistrator @register

**Comment by ablaom** [comment]
@JuliaRegistrator register

release notes:

- (**enhancement**) Allow specification `Callback(f, ..., raw=true)` to force `f` to be applied to the raw model object `model`, instead of `expose(model)` (#22)

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

release notes:

- [ ] (**enhancement**) Add the controls `NumberCount(f)`, `Loss(f)` and `TrainingLosses(f)`, which apply `f` to the control cycle count, the current loss, and the current batch of training losses, respectively. Specify `stop_when_true=true` to trigger a stop when `f` returns `true`.

**Comment by ablaom** [comment]
@JuliaRegistrator register

release notes:

- [x] (**breaking**) Rename`NumberCount` to `WithNumberDo`, `Loss` to `WithLossDo`, and `TrainingLoss` to `WithTrainingLossDo`.

**Comment by ablaom** [comment]
@JuliaRegistrator register

release notes:

- [x] (**bug fix**) Fix `IterationControl.skip` (#32)

**Comment by ablaom** [comment]
@JuliaRegistrator register

release notes:

- [x] Bump EarlyStopping compat to add `NumberSinceBest` stopping criterion.

**Comment by ablaom** [comment]
@JuliaRegistrator register

release notes:

- [x] Don't lower  `verbosity` in `train!(...)` when passing to `update` calls. Breaks logging behaviour for controls but built-in controls have been fixed so that their use in `train!(...)` is unaffected. (#34)

**Comment by ablaom** [comment]
@JuliaRegistrator register

release notes:

- [x] Fix a typo in logging: `Stepping ...` instead of `Steping...` (#36)

**Comment by ablaom** [comment]
@JuliaRegistrator register

release notes:

- bump [compat] for EarlyStopping to 0.1.8

**Comment by ablaom** [comment]
@JuliaRegistrator register

release notes:

- Add some extra data to the `state` of some controls, and make this information user-inspectable (via the return value of `train!`, as generated by the control's `takedown` method. In particular, make add the loss to the `WithLossDo` control `state` and the latest training loss available to the `WithTrainingLossesDo` control state (#39)
- Tweak some logging
- At the default logging level of `verbosity=1`, log to `Info` the final loss, and the final training loss, when available (resolves part of #38)
- Add a new control wrapper `IterationControl.louder(control, by=1)` to increase the verbosity of `control` (the value of `by` is added to the `verbosity` level, as passed to `IterationControl.train!(..., verbosity=....)`).

**Comment by ablaom** [comment]
@JuliaRegistrator register

release notes:

- [x] (**breaking**) Change the meaning of "number" in `WithNumberDo` to be the control cycle count, rather than the number of applications of that particular control (#38, PR #44).

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master
